database/index.js

```diff
@@ -0,0 +1,30 @@
+require('dotenv').config();
+const Item = require('./itemSchema');
+
+const mongoose = require('mongoose');
+mongoose.connect(`mongodb+srv://${process.env.DB_USER
+}:${process.env.DB_PASS}@cluster0-cmnav.mongodb.net/test?retryWrites=true&w=majority`, {useNewUrlParser: true, dbName : 'seabay'});
+
+//test interface
+const Cat = mongoose.model('Cat', { name: String });
+
+const kitty = new Cat({ name: 'Zildjian' });
+kitty.save().then(() => console.log('meow'));
+
+const addItem = (id, name, price, category, images) => {
+  return new Item({
+    id,
+    name, 
+    price, 
+    category,
+    images
+  }).save()
+}
+
+const getAll = () => {
+  const query = Item.find({})
+  const promise = query.exec()
+  return promise;
```

Comment:
I believe you can instead do this as well if you want the Promise.

`return Item.find({}).exec();`

 However, I returned just the `Query` instance and was fine.

Reply:
Good suggestion! I made that change